README.md

```diff
@@ -84,7 +84,12 @@ Tests are an important part of the development and evolution of this project,
 and new contributions are expected to include tests for any functionality
 change.  To run the tests, pass the `test` verb to the `bootstrap` script:
 
-    ./Utilities/bootstrap --build-tests test
+    ./Utilities/bootstrap \
+        --swiftc /path/to/swift-linux-x86_64/bin/swiftc \
```

Comment:
does it have to be linux path `/path/to/swift-linux-x86_64` or can we make it generic like
`path/to/snapshot/usr/bin/swiftc`